### ISSUE TITLE
Support PostgreSQL

### ISSUE THREAD

**Issue by Half-Shot** [turt2live/matrix-trello-bot#28]
Looks like [Sequelize](https://www.npmjs.com/package/sequelize) already can

**Comment by turt2live** [comment]
This is highly unlikely because the bot doesn't store much data and the config needs to be mounted anyways.

**Comment by Half-Shot** [comment]
This is for kubes / hhs where we can't store any persistent data without doing a lot of faffing about, whereas readonly data is somewhat easier to mount. I'll probably look into doing this feature when we get to it.

**Comment by turt2live** [comment]
I'd much rather the bot store data within matrix instead of a database. We don't currently have secure storage capabilities though.

For such a small bot, supporting 2 database engines is not ideal. Similarly, supporting a migration is not ideal for a project this size.

If a second engine is to be supported, it'll be matrix instead.

**Comment by Half-Shot** [comment]
How would you store tokens for users using Matrix, in theory? The bot's account data?

**Comment by turt2live** [comment]
yup, using a system similar to key backups (if not key backups themselves)
